api/v1alpha/src/lib/config.js

```diff
@@ -19,9 +19,9 @@
 var config = {};
 
 config.verboseMode = false;
-config.productName = 'CDS';
+config.productName = 'Datashare';
 config.productVersion = '0.0.1';
-config.cdsManagedLabelKey = 'cds_managed';
+config.cdsManagedLabelKey = 'datashare_managed';
```

Comment:
Is there a migration path for customers that are using the old label key? i.e. Do we need to support both label keys?

Reply:
Until we publish a v1 public on marketplace I wasn't planning to add any data migration. There's more than just this, Ie table schema changes etc.

Reply:
ack. We can circle back for a migration strategy for v1.